devel/python/python/ert_gui/ertwidgets/models/selectable_list_model.py

```diff
@@ -0,0 +1,46 @@
+from PyQt4.QtCore import QObject, pyqtSignal
+
+
+class SelectableListModel(QObject):
```

Comment:
I agree. My goal in this round, however, is removing dependency on ErtConnector. So this is a future task.